src/archivematicaCommon/lib/archivematicaFunctions.py

```diff
@@ -535,3 +557,9 @@ def relative_path_to_aip_mets_file(uuid, current_path):
     mets_name = "METS.{}.xml".format(uuid)
     mets_path = "{}/data/{}".format(package_name_without_extensions, mets_name)
     return mets_path
+
+
+def chunk_list(input_list, chunk_size=10):
+    """Yield successive chunks of size chunk_size from input list."""
+    for index in range(0, len(input_list), chunk_size):
+        yield input_list[index : index + chunk_size]
```

Comment:
I'm wondering if we should write something that looks more like [`more_itertools.grouper`](https://github.com/more-itertools/more-itertools/blob/e899af25e95a2300d63b832f8a7861d55b979371/more_itertools/recipes.py#L284-L297) since `zip_longest` is written in C and it looks like it could work with any sequence.

Reply:
Oh nice! Yeah I can do that for sure.